src/sections/footer.html

```diff
@@ -0,0 +1,20 @@
+
+<div class="bg-white">
+    <div class="bg-primary bg-gradient-to-r from-third to-secondary/40 
+        lg:[clip-path:polygon(0_0,100%_35%,100%_100%,0_100%)] md:h-32 h-full">
+
+        <div class="md:flex grid place-items-center md:items-center justify-evenly pt-0 md:pt-10 pl-3 md:pl-0 gap-0 md:gap-10">
+            <div class="cursor-pointer">
+                <img src="/src/assets/Logo.svg" class="w-28 md:w-32" alt="Ojemba's logo">
+            </div>
+            <div>
+                <ul class="leadig-3 md:flex grid text-white font-roboto gap-10 cursor-pointer">
+                    <li>Impressum</li>
+                    <li>Datenschutz</li>
+                    <li>FAQ</li>
+                    <li>Kontakt</li>
+                </ul>
+            </div>
+        </div>
```

Comment:
Hello @uwumukiza123, Nice job so far 👌 but unfortunately there are some changes you still need to work on to improve the layout of your design to match Figma design. I have attached both layouts for more details

### Figma design
![Footer](https://github.com/kaitcham/ojemba/assets/42999621/16ba5686-1a37-42d4-8fea-2c29209fb460)
### Your Design
![Footer design](https://github.com/kaitcham/ojemba/assets/42999621/945904b3-edf7-44fa-86da-48ec23200558)